docs/_includes/header.html

```diff
@@ -4,7 +4,10 @@
     {%- assign default_paths = site.pages | map: "path" -%}
     {%- assign page_paths = site.header_pages | default: default_paths -%}
     {%- assign titles_size = site.pages | map: 'title' | join: '' | size -%}
-    <a class="site-title" rel="author" href="{{ "/" | relative_url }}">{{ site.title | escape }}</a>
+    <a class="site-title" rel="author" href="{{ "/" | relative_url }}">
+      <img src="../images/bookmarkicon.png" height="80" width="50"/>
+           {{ site.title | escape }}
+    </a>
```

Comment:
This is the **attempt** at creating an icon beside _bookmark_ if it doesn't work I can revert this header.html file